dev/github_jira_sync.py

```diff
@@ -122,13 +129,13 @@ def reset_pr_labels(pr_num, jira_components):
     url = '%s/issues/%s/labels' % (GITHUB_API_BASE, pr_num)
     labels = ', '.join(('"%s"' % c) for c in jira_components)
     try:
-        request = urllib2.Request(url, data='{"labels":[%s]}' % labels)
+        request = Request(url, data=('{"labels":[%s]}' % labels).encode('utf-8'))
```

Comment:
@WeichenXu123, can use `u'{"labels":[%s]}' % labels).encode('utf-8')` so that it's always an unicode in both Python 2 and Python 3, and always executes same code path?

Currently,
- In Python 2, it will be bytes -> unicodes -> bytes (by implicit conversion)
- in Python 3, unicodes -> bytes.

After using `u`, 

Both will do unicodes -> bytes.

Reply:
Ah, hm, there are multiple places like this.
Okay, NVM. Components won't likely be out of ascii (which might be happen during implicit conversion bytes -> unicodes in Python 2)